src/Resources/config/services.xml

```diff
@@ -16,6 +16,7 @@
         <service id="webpack_encore.tag_renderer" class="Symfony\WebpackEncoreBundle\Asset\TagRenderer">
             <argument type="service" id="webpack_encore.entrypoint_lookup_collection" />
             <argument type="service" id="assets.packages" />
+            <argument type="string" id="webpack_encore.crossorigin" />
```

Comment:
`id` is not a valid attribute for `<argument type="string">` (and we generally omit `type="string"` entirely as that's the default behavior too)

Reply:
I think I have done this right now, just `<attribute>anonymous</attribute>` ???